packages/uniforms-material/src/LongTextField.tsx

```diff
@@ -1,26 +1,29 @@
-import React, { ReactNode } from 'react';
+import React from 'react';
 import TextField, { StandardTextFieldProps } from '@material-ui/core/TextField';
-import { FieldProps, connectField, filterDOMProps } from 'uniforms';
+import { connectField, FieldProps, filterDOMProps } from 'uniforms';
```

Comment:
Is your IDE configured to do that? I _still_ didn't managed to configure that, but please, keep them ordered.

Reply:
Yeah, I fixed 4 of 5 exactly the same rearrangements in other files, this one escaped me.